docs/sources/datasources/cloudwatch.md

```diff
@@ -381,7 +381,19 @@ It's now possible to configure data sources using config files with Grafana's pr
 
 Here are some provisioning examples for this data source.
 
-### Using credentials profile name (non-default)
+### Using AWS SDK Default
+
+```yaml
+apiVersion: 1
+datasources:
+  - name: CloudWatch
+    type: cloudwatch
+    jsonData:
+      authType: default
+      defaultRegion: eu-west-2
+```
+
+### Using credentials profile name (non-default) 
```

Comment:
How about "Using profile name of the credential" or "Using credential's profile name"?